### ISSUE TITLE
New posts randomly are not "saved" / does not apper anywhere

### ISSUE THREAD

**Issue by kristapsk** [damus-io/damus#767]
Not sure how to reproduce or debug this. But happened longer time ago with a reply (tried replying there at least twice), once some two days ago with a simple new post and today again when I replied to post. After pressing "Post" Damus acts like everything is ok, but it does not appear anywhere. 

Using version 1.1.0 from App Store.

**Comment by alltheseas** [comment]
@kristapsk how is this working for you now?

**Comment by kristapsk** [comment]
@alltheseas Don't have TestFlight access to Damus, will need to wait for the next App Store release to test.

**Comment by kristapsk** [comment]
But I haven't noticed posts not being saved, apart from reposts, recently.

**Comment by alltheseas** [comment]
> @alltheseas Don't have TestFlight access to Damus, will need to wait for the next App Store release to test.

AppStore now has latest version 1.4.2 (2)

**Comment by kristapsk** [comment]
Yes, I'm running it currently, see https://github.com/damus-io/damus/issues/841#issuecomment-1509757168.

**Comment by kristapsk** [comment]
Randomly having these problems again, all cases where either when replying or reposting with comment ("quote"). Version 1.5.

**Comment by kristapsk** [comment]
I think I noticed one pattern - I try to post, it doesn't appear. When I navigate to my profile, to check, it's empty. I decide app has in bad state and close it and restart. But if I instead just keep it open longer, sometimes for minute, sometimes for ten, then eventually it is posted. But there is no feedback about that, it just appears in feed at some point.

**Comment by jb55** [comment]
On Sat, Aug 12, 2023 at 11:41:52AM -0700, Kristaps Kaupe wrote:
>I think I noticed one pattern - I try to post, it doesn't appear. When I navigate to my profile, to check, it's empty. I decide app has in bad state and close it and restart. But if I instead just keep it open longer, sometimes for minute, sometimes for ten, then eventually it is posted. But there is no feedback about that, it just appears in feed at some point.

Internally we have a "Postbox" that continuously attempts to send a post
until it gets a success response. If it keeps failing for whatever
reason you may not see it in your feed until it is posted. It would be
nice if we had an pending "outbox" view but this is pretty niche.

**Comment by kristapsk** [comment]
Today I'm noticing such problems again, but there is another thing - Damus has problems connecting to wss://relay.damus.io (other relays work).

**Comment by alltheseas** [comment]
> Today I'm noticing such problems again, but there is another thing - Damus has problems connecting to wss://relay.damus.io (other relays work).

It's a separate issue https://github.com/damus-io/damus/issues/1613

**Comment by kristapsk** [comment]
> > Today I'm noticing such problems again, but there is another thing - Damus has problems connecting to wss://relay.damus.io (other relays work).
> 
> It's a separate issue #1613

Yeah, but it should not cause this, Damus should be working well even without Damus relay configured, right?

**Comment by alltheseas** [comment]
> it should not cause this

Can you specify what "this" behavior you observe? Can you remind me your iOS, and Damus versions?

>  Damus should be working well even without Damus relay configured, right?

Correct.

**Comment by kristapsk** [comment]
> Can you specify what "this" behavior you observe? Can you remind me your iOS, and Damus versions?

Just randomly posts I wrote aren't saved / broadcast, they just not appear anywhere.

iOS 15.7.9 (I have iPhone 7, so no iOS 16+ for me) and Damus 1.5 (8) (should be latest App Store version).

**Comment by alltheseas** [comment]
> should be latest App Store version

Let's test again when 1.6 AppStore version comes out 🙏

**Comment by alltheseas** [comment]
@kristapsk are you still experiencing this issue?

Might a design/UI affordance that signifies "pending" notes help with this issue, which centers around lack of feedback on hitting the post button?

**Comment by kristapsk** [comment]
> @kristapsk are you still experiencing this issue?
> 
> Might a design/UI affordance that signifies "pending" notes help with this issue, which centers around lack of feedback on hitting the post button?

As mentioned above, I have iPhone 7 with iOS 15, so I'm stuck with old Damus version, can't upgrade and test anything.

**Comment by alltheseas** [comment]
Will revisit upon iOS 15 nav implementation #1929

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
